packages/common/test/viewport_scroller_spec.ts

```diff
@@ -23,18 +23,21 @@ import {BrowserViewportScroller, ViewportScroller} from '../src/viewport_scrolle
   describe('BrowserViewportScroller', () => {
     let scroller: ViewportScroller;
     let documentSpy: any;
-    beforeEach(() => {
-      documentSpy = jasmine.createSpyObj('document', ['querySelector']);
-      scroller = new BrowserViewportScroller(documentSpy, {scrollTo: 1}, null!);
-    });
-    it('escapes invalid characters selectors', () => {
-      const invalidSelectorChars = `"' :.[],=`;
-      // Double escaped to make sure we match the actual value passed to `querySelector`
-      const escapedInvalids = `\\"\\' \\:\\.\\[\\]\\,\\=`;
-      scroller.scrollToAnchor(`specials=${invalidSelectorChars}`);
-      expect(documentSpy.querySelector).toHaveBeenCalledWith(`#specials\\=${escapedInvalids}`);
-      expect(documentSpy.querySelector)
-          .toHaveBeenCalledWith(`[name='specials\\=${escapedInvalids}']`);
+    describe('scrollToAnchor', () => {
```

Comment:
While we are here, let's add some empty lines between describe and it blocks, to keep it more inline with other files in the repo.

```suggestion

    describe('scrollToAnchor', () => {
```

Reply:
You could also remove the duplicated copyright block above, and in fact we might as well remove the `{ .. }` block around the whole set of tests, since this is from a long time back when we were not using a bundler before loading the files into the browser.

Reply:
Yeah better! I wasn't sure about the code style so I left it as it was in the file.